src/index.html

```diff
@@ -7,8 +7,65 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header__wrap"
+      id="home"
+    >
```

Comment:
fix your codestyle according to [checklist](https://github.com/mate-academy/layout_moyo-header/blob/master/checklist.md) and fix it everywhere
```suggestion
    <header class="header__wrap" id="home">
```